content/uiux/concepts/crazy-eights/crazy-eights.md

```diff
@@ -0,0 +1,13 @@
+---
+Title: 'Crazy Eights'
+Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Crazy Eights's** a Design Sprint method: meaning it is a quick sketching exercise that challenges people to sketch eight different ideas in eight minutes. The goal is to push beyond your first idea, which is usually basic(the least innovative), and to generate a wide variety of solutions to your initial design.
+
+#### Example  
+To properly implement the crazy eight method would be to 1st begin with an idea in mind for your design then setting a 8minute timer and folding a piece of paper 3 times so that you have eight boxes you then start the timer begin sketching and spend no more than 1 minute per box. Do this with other members in your team the more the ideas the better since this is just the drafting stage.
+![crazy 8's example #1](<www.github.com/codecademy/docs/media/crazy8's #1.jpg>) ![8minTimer](www.github.com/codecademy/docs/media/8mintimer.jpg) ![crazy 8's example #2](<www.github.com/codecademy/docs/media/crazy8's #2.png>)
```

Comment:
I think only one of the crazy 8's example image should be enough. Please remove the other image and the `8mintimer.jpg` image. Also remember to remove these images from the media folder.

Reply:
done